pkg/controller/kubedirectorcluster/members.go

```diff
@@ -280,10 +317,65 @@ func handleReadyMembers(
 				)
 				return
 			}
+
+			memberVersion := *m.StateDetail.LastConnectionVersion
+
+			shared.LogInfo(
+				reqLogger,
+				cr,
+				shared.EventReasonCluster,
+				fmt.Sprintf("memberVersion for pod %s is %d", m.Pod, memberVersion),
+			)
+
+			if memberVersion < connectionsVersion {
+				shared.LogInfo(
```

Comment:
so if the annotation is deleted then we will never come here. Can we infer connectionsVersion from members ?